packages/components/organisms/f-header/src/components/Header.vue

```diff
@@ -173,6 +174,10 @@ export default {
             return this.headerBackgroundTheme === 'transparent' && !this.mobileNavIsOpen;
         },
 
+        showSearchHeader () {
```

Comment:
I'm not sure if we should add such domain-specific language here. I understand its solving a search problem, but is it possible to re-word it to be a little more generic/reusable?

Reply:
happy to rename it, but I'm not sure what could be a good name for this theme. It was confirmed by the design team that this variant will be used only on search.

Reply:
Agree with @jamieomaguire – I'd name it closer to how the theme looks than the page it's used on. Just as it's only used on Search now, doesn't mean that it'll never be on another page in the future.

Is the main difference in the visual theme that it's orange on narrow screens and white on wide screens?

Reply:
It's white on narrow now, and transparent with no gradient or `box-shadow` on wide screens.

Reply:
I'd probably call it something like `disappearingWhite`. As long as you document what it does on narrow and wide in the prop for the theme, then that should be good then.